spec.html

```diff
@@ -25027,6 +25027,16 @@ <h1>Runtime Semantics: SymbolDescriptiveString ( _sym_ )</h1>
         </emu-clause>
       </emu-clause>
 
+      <emu-clause id="sec-symbol.prototype.description">
+        <h1>get Symbol.prototype.description</h1>
+        <p>`Symbol.prototype.description` is an accessor property whose set accessor function is *undefined*. Its get accessor function performs the following steps:</p>
+        <emu-alg>
+          1. Let _s_ be the *this* value.
+          1. Let _sym_ be ? thisSymbolValue(_s_).
```

Comment:
You could also do

```
Let _sym_ be ? thisSymbolValue(*this* value).
```

Reply:
The reigning convention seems to be to have it in a separate variable; i don’t have a strong opinion either way as long as we’re consistent.